domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Controllers/JsonDefault.cs

```diff
@@ -0,0 +1,13 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace API.IntegrationTests.Controllers;
+
+public static class JsonDefault
+{
+    public static JsonSerializerOptions Options = new JsonSerializerOptions
```

Comment:
Can you make this a static get-property? Currently it is possible for the tests to assign a new value to `Options`